testdata/ansible/memcached-operator/Makefile

```diff
@@ -48,7 +48,7 @@ endif
 
 # Set the Operator SDK version to use. By default, what is installed on the system is used.
 # This is useful for CI or a project to utilize a specific version of the operator-sdk toolkit.
-OPERATOR_SDK_VERSION ?= v1.30.0
+OPERATOR_SDK_VERSION ?= v1.28.0
```

Comment:
Not sure why this is not being updated to `1.31.0` automatically by the release script.

Reply:
Yeah, odd that it's updating it backwards...

Reply:
@rashmigottipati this version is taken from the binary that is available locally, (ie) here: https://github.com/operator-framework/operator-sdk/blob/c6d1bf979e86dc0c055060e3872f5e7bbd629f3f/internal/plugins/manifests/v2/init.go#L72

Which is again obtained from here: https://github.com/operator-framework/operator-sdk/blob/c6d1bf979e86dc0c055060e3872f5e7bbd629f3f/Makefile#L9

We can leave this to 1.30.0 (which is the latest available one), till we release 1.31. @rashmigottipati you could fetch all the latest available tags (could run `git fetch --all`), and generate the testate again. That should bump it back to 1.30.

Reply:
Thanks @varshaprasad96, just fixed it :) It should be updated now